event_dispatcher.rst

```diff
@@ -246,6 +246,73 @@ there are some minor advantages for each of them:
 * **Listeners are more flexible** because bundles can enable or disable each of
   them conditionally depending on some configuration value.
 
+.. _event-aliases:
```

Comment:
This can be removed as it is not used and the anchor does not differ from the current headline.